docs/guides/admin/docs/upgrade.md

```diff
@@ -28,10 +28,10 @@ Please make sure to compare your configuration against the current configuration
 Install and configure a standalone Elasticsearch node
 -----------------------------------------------------
 
-In the past, Opencast came with its own integrated Elasticsearch node. However, recent versions of Elasticsearch no longer
-support to be embedded in applications. Since the Elasticsearch client was updated to version 7, Opencast now requires an
-external Elasticsearch node of the same version to be present. This means, that all Opencast adopters now have to run
-Elasticsearch.
+Although Opencast has come with its own integrated Elasticsearch node in the past, recent versions of Elasticsearch no
+longer support being embedded in applications.
+Since the Elasticsearch client was updated to version 7, Opencast requires an external Elasticsearch node of the same
+version to be present. This means all Opencast adopters now have to run Elasticsearch as a service.
 
 Please check [the documentation](modules/searchindex/elasticsearch.md) for information about how to setup an external node.
```

Comment:
Link likely isn't up to date with the updated ES docs pr that's still outstanding.